gnomad_qc/v5/resources/annotations.py

```diff
@@ -213,18 +221,19 @@ def get_freq(
 ######################################################################
 
 
-def get_info_ht(test: bool = False) -> VersionedTableResource:
+def get_info_ht(test: bool = False, environment: str = "rwb") -> VersionedTableResource:
```

Comment:
this line (https://github.com/broadinstitute/gnomad_qc/blob/e86e566cfb1d58068158d34ae2e1ab2ac84b673e/gnomad_qc/v5/annotations/generate_variant_qc_annotations.py#L236) also should be updated